packages/app/src/components/Label.tsx

```
@@ -0,0 +1,15 @@
+
+export default function Label({
+  className = "",
+  children,
+}: {
+  className?: string;
+  children: React.ReactNode;
+}) {
+  return (
+    <label className={twMerge("block mt-5 mb-1 text-sm opacity-80", className)}>
+      {children}
+    </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)